Projects/PaywallTester/Project.swift

```diff
@@ -23,7 +23,7 @@ let allDeploymentTargets: DeploymentTargets = .multiplatform(
 var additionalFiles: [FileElement] = [
     .glob(pattern: "../../Global.xcconfig")
 ]
-if FileManager.default.fileExists(atPath: "../../Local.xcconfig") {
+if FileManager.default.fileExists(atPath: "Local.xcconfig") {
     additionalFiles.append(.glob(pattern: "../../Local.xcconfig"))
 }
```

Comment:
Maybe not for this PR, but would it be possible/make sense to have this be reused from the main `Project.swift` instead of repeated in every `Project.swift` file?
Maybe something similar to `xcconfigFileConfigurations` below. E.g. `commonFiles`.

Reply:
I think so, let me re-do this

Reply:
I'll tackle this in another PR, but I think that since we're adding those files to the workspace it should be ok if we remove them from each project.